docs/CeedlingPacket.md

```diff
@@ -985,6 +985,40 @@ that you execute on target hardware).
 
   **Default**: FALSE
 
+  ---
+
+  **Note:**
+
+    The configuration can be combined together with
+    
+    ```
+    :test_runner:
+        :cmdline_args: true
+    ```
+
+    After setting **cmdline_args** to **true**, the debuger will execute each of test
+    case from crashing test file separately. The exclude and include test_case patterns will
+    be applied, to filter execution of test cases.
+
+    The .gcno and .gcda files will be generated and section of the code under test case
+    causing segmetation fault will be omitted from Coverage Report.
+
+    The default debugger (**gdb**)[https://www.sourceware.org/gdb/] can be switch by any other
+    via setting new configuration under tool node in project.yml. At default set to:
+
+    ```
+      :tools:
+        :backtrace_settings:
+          :executable: gdb
+          :arguments: [ "-q --eval-command run --eval-command backtrace --batch --args" ]
+    ```
+    
+    Important. The debugger which will collect segmentation fault should run in:
+    - background
```

Comment:
I think this needs a new line of top or some sort of indentation to be regarded as list.

```suggestion

    - background
```

Reply:
Hi @Letme,

True, adding extra space fix the documentation. Thank you for spotting this.

The line was updated.